.docker/Crawler/Dockerfile

```diff
@@ -10,6 +10,4 @@ ADD ./run-crawler.sh /Arriba/Tools/app
 WORKDIR /Arriba/Tools/app
 RUN chmod +x ./run-crawler.sh
 
-RUN ls -l
-RUN find /Arriba
-CMD ["./run-crawler.sh", "configName=default", "mode=-i"] 
\ No newline at end of file
+CMD ["./run-crawler.sh", "configName=default", "mode=-i"]
```

Comment:
The documentation says that if we pass some argument to the docker run it will override the CMD (completely), and if we use ENTRYPOINT and try to pass a new command to the Docker run it will be handle as argument for the command. 
The only way to override the entrypoint is using the --entrypoint.

If we do something like

'''
ENTRYPOINT ["./run-crawler.sh"]
CMD["configName=default", "mode=-i"]
'''

By default when we run.. Docker run crawler it will execute ./run-crawler.sh configName=default mode=-i

and if we do ./docker run crawler mode=foo configName=bar, it will override the CMD and use what we are specifying.

Reply:
fixed